### ISSUE TITLE
Use Public Image Instead of Building One

### ISSUE THREAD

**Issue by jamesward** [GoogleCloudPlatform/cloud-run-button#181]
If a project publishes a public container image then maybe the button should allow the skipping the build and just using the existing image.  Via `app.json` like:
```
{
    "build": {
        "skip": false
    },
    "override": {
        "image": "gcr.io/cloudrun/hello"
    }
}
```

Not sure if this is a good idea or not.

**Comment by ahmetb** [comment]
This basically makes the project an alternative to deploying directly from Cloud Run console. Similarly, broadens the focus from "deploy repos" to "deploy images or repos".

**Comment by jamesward** [comment]
Yeah, maybe if https://console.cloud.google.com/run/create had a way to specify some default params (image, service name, allow-unauthenticated, etc) then that'd be a better way to go.  Still would be missing env var handling though.
